invoke-ai/InvokeAI
# fix(ui): reset page when search term changes

## Summary

When changing the search term, we should reset the page (set offset to 0). 

## Related Issues / Discussions

https://discord.com/channels/1020123559063990373/1149513625321603162/1257795535482392727

## QA Instructions

- Without a search term set, select a page in the gallery
- Enter a search term that will result more than 1 page of results

Without this PR, we expect that the search results will start on a page equal to the previously selected page, or the last possible page of the search results.

With this PR, we expect that the search results will start on the first page.

## Merge Plan

n/a

## Checklist
